con_knative-OCP-311.adoc

```diff
@@ -0,0 +1,22 @@
+// This assembly is included in the following assemblies:
+//
+// assembly_knative-OCP-311.adoc
+
+[id='knative-ocp-311_{context}']
+= Knative on an OpenShift cluster
+
+== Developer Preview 0.3.0
+
+NOTE: The functionality introduced by Knative on an OpenShift cluster is developer preview only. Red Hat support is not provided, and this release should not be used in a production environment.
+
+All of the required components of Knative on an OpenShift cluster, including software dependencies and configurations, can be installed by using the script provided by Red Hat in the OpenShift Cloud Functions `knative-operators` repository. The script installation method is recommended.

```

Comment:
I'd probably remove the "in the OpenShift Cloud Functions `knative-operators` repository" part of the second sentence now, since the code may move into a different repo, and being too specific about this just means more maintenance (we include a link in the instructions anyway for cloning the repo) - should apply anywhere this statement is used.